sig-and-wg/sig-core/proposals/installer-release.md

```diff
@@ -0,0 +1,29 @@
+# Kyma Installer with embedded kyma charts and tiller
+
+Created on 2018-09-19 by Piotr Bochyński (@pbochynski).
+
+## Status
+
+Proposed on 2018-09-19
+
+## Abstract
+
+Current installation process involves downloading kyma charts from cloud storage. Such solution allows to release kyma installer and kyma charts independently. In the consequence, there is possibility to install different versions (or flavors) of kyma components using single installer binary. It seems to be a nice feature, but in the reality introduces problems:
+- Build/Release process is more complex because it has to publish kyma charts in cloud storage.
+- It is possible to run installer with kyma charts version not supported (not tested with the installer in given version).
+- Upgrade process is more complex (usually installer has to be upgraded and then kyma release URL has to be updated)
+
+## Proposal
+
+Installer will contain kyma charts and scripts in the docker image. The build process has to be adapted to produce installer image for every kyma snapshot/release. It is important to include also version.env file in the image as it is essential part of kyma charts.
```

Comment:
`Kyma` - capitalize
`Docker` - capitalize

`The build process must be adapted to produce the installer image for every Kyma snapshot or release.`
``It is also important to include the `version.env` file in the image as it is an essential part of Kyma charts.``